app/frontend/src/components/apiAccess/Step1.vue

```diff
@@ -70,23 +79,25 @@
 <script>
 import { mapGetters, mapMutations } from 'vuex';
 
+import ClientStatus from '@/components/apiAccess/ClientStatus.vue';
 import { KcClientStatus, KcEnv } from '@/utils/constants';
-import { buildClientStatusSpan } from '@/utils/util.js';
 
 export default {
   name: 'ApiAccessStep1',
+  components: {
+    ClientStatus
+  },
   data() {
     return {
       KcClientStatus: KcClientStatus,
-      env: KcEnv
+      envs: KcEnv
```

Comment:
These should probably be computed as they are meant to be constants.

Reply:
Fixed